app/src/main/java/com/bnyro/contacts/ui/screens/SettingsScreen.kt

```diff
@@ -67,6 +70,16 @@ fun SettingsScreen(onDismissRequest: () -> Unit) {
                         themeModel.themeMode = ThemeMode.fromInt(it)
                     }
                 }
+                SettingsContainer {
+                    SettingsCategory(title = stringResource(R.string.default_account))
+                    ListPreference(
+                        preferenceKey = Preferences.defaultAccountKey,
+                        title = R.string.default_account,
+                        entries = contactsModel.getAvailableAccountNames(),
+                        values = contactsModel.getAvailableAccountTypes(),
+                        defaultValue = contactsModel.getAvailableAccountTypes().first()
+                    )
+                }
```

Comment:
I don't really like this approach since the same accounts would be used by default for `local` and `device` contacts although they use different account names in general.